install/helm/open-match/templates/service-account.yaml

```diff
@@ -34,16 +34,27 @@ metadata:
     app: {{ template "openmatch.name" . }}
     release: {{ .Release.Name }}
 rules:
+# Define om-service-role to use om-core-podsecuritypolicy
 - apiGroups:
   - extensions
   resources:
   - podsecuritypolicies
   resourceNames:
-  - om-podsecuritypolicy
+  - om-core-podsecuritypolicy
   verbs:
   - use
+# Grant om-service-role get & list permission for k8s endpoints and pods resources
+# Required for e2e in-cluster testing.
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
```

Comment:
Do all jobs need this or just a few. Create a new Service Account if the elevation isn't universal. There's things like the demo and swaggerui that probably don't need this.